src/templates/mht_document.tpl

```diff
@@ -0,0 +1,22 @@
+From: "Generated by html2docx lib"
+Subject: Html To Docx Document
+Date: Thu, 6 Aug 2015 02:49:38 -0700
```

Comment:
Could you check if all these fields are mandatory? Maybe we can skip inserting this information if it is not needed.